scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -191,7 +192,7 @@ class PostgresStorage(
         @Suppress("TooGenericExceptionCaught")
         return try {
             val scanResults = runBlocking(Dispatchers.IO) {
-                packages.chunked(max(packages.size / 50, 1)).map { chunk ->
+                packages.chunked(max(packages.size / LocalScanner.NUM_STORAGE_THREADS, 1)).map { chunk ->
```

Comment:
As this will solve our issue we have at the moment, I'm ok with the change.

But I'm not completely sure what the advantage is to chunk the packages here.
Isn't the database optimized to to parallelize the query?
Additionally it would reduce the complexity and the possibility of issues to not chunk the requests.

Reply:
@MarcelBochtler The reason was that in all tests I did fetching scan results in chunks was significantly faster than fetching them in a single request.